### PR TITLE
Lovelace view conf icon prev

### LINKED ISSUE #5734: Lovelace view configuration does not show icon preview
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

Lovelace view configuration does not show icon preview.

image

## Expected behavior

To show the icon that is currently set, as is done in many other places.

E.g., Helpers:

image

## Environment

- Home Assistant release with the issue: Home Assistant 0.110.0.dev20200502
- Last working Home Assistant release (if known): unkown
- Browser and browser version: Latest chrome
- Operating system: macOS

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

### PR DESCRIPTION
**Pull request by matstrange**
## Proposed change

Fixes issue #5734 where the Lovelace View Configuration does not preview the icon that has been input, which is a different behaviour to other places in the UI.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #5734
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [x] Tests have been added to verify that the new code works.

lovelace view icon fix
Icon now shows as expeced as soon as a valid icon is entered

lovelace view icon fix save test
And after saving the entry it persists as expected.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/editor/view-editor/hui-view-editor.ts [hunk 1]
```diff
@@ -93,16 +95,18 @@ export class HuiViewEditor extends LitElement {
...
+        <ha-icon-input
           .label="${this.hass.localize(
             "ui.panel.lovelace.editor.card.generic.icon"
-          )}  (${this.hass.localize(
+          )} (${this.hass.localize(
             "ui.panel.lovelace.editor.card.config.optional"
           )})"
           .value=${this._icon}
+          .placeholder=${this._icon ||
+          stateIcon(this.hass.states[this._entity])}
```
Comment (bramkragten) [inline src/panels/lovelace/editor/view-editor/hui-view-editor.ts:106]: A view doesn't have an entity?

Comment (bramkragten) [inline src/panels/lovelace/editor/view-editor/hui-view-editor.ts:106]:
```suggestion
          .placeholder=${this._icon}
```

File: src/panels/lovelace/editor/view-editor/hui-view-editor.ts [hunk 2]
```diff
@@ -11,6 +11,8 @@ import {
 import { fireEvent } from "../../../../common/dom/fire_event";
 import { slugify } from "../../../../common/string/slugify";
 import "../../../../components/ha-switch";
+import "../../../../components/ha-icon-input";
+import { stateIcon } from "../../../../common/entity/state_icon";
```
Comment (bramkragten) [inline src/panels/lovelace/editor/view-editor/hui-view-editor.ts:15]:
```suggestion
```

File: src/panels/config/integrations/ha-config-integrations.ts [hunk 3]
```diff
@@ -619,6 +619,9 @@ class HaConfigIntegrations extends SubscribeMixin(LitElement) {
         .ignored {
           border: 1px solid var(--light-theme-disabled-color);
         }
+        .ignored img {
+          filter: grayscale(1);
+        }
```
Comment (bramkragten) [inline src/panels/config/integrations/ha-config-integrations.ts:624]: You should rebase, as this is from your other PR

Comment (matstrange) [reply src/panels/config/integrations/ha-config-integrations.ts:624]: Sorry I tried to do that but must have done it wrong :( 

I've tested your suggestions locally and they work perfectly. Is it best to accept via the 'commit suggestion' link or try to start fresh and re-upload?

image

Comment (bramkragten) [reply src/panels/config/integrations/ha-config-integrations.ts:624]: It is easier to start fresh :-)

**Comment by matstrange** [comment]
Closing, and submitting a new clean PR

**Pull request closed without merging**